osmcrops.csv

```diff
@@ -1,77 +1,77 @@
+Cashewapple,cashew_nut,cashew_trees,orchard,
+Tung nuts,nut,,orchard,Too unspecific?
+Pears,pear,pear_trees,orchard,
+Tea,tea,tea_plants,orchard,
+Cherries,cherry,cherry_trees,orchard,
+Cherries- sour,cherry,cherry_trees,orchard,
+Mangoes- mangosteens- guavas,guava|mango|mangosteen,guava_trees|mango_trees|mangosteen_trees,orchard,
+Papayas,papaya,papaya_trees,orchard,
+Pineapples,pineapple,pineapple_plants,orchard,
+Agave fibres nes,agave,agave_plants,orchard,"source data only has fibres, but that does not matter"
+Avocados,avocado,avocado_trees,orchard,
+Figs,fig,fig_trees,orchard,
+Lemons and limes,lemon|lime,lemon_trees|lime_trees,orchard,
+Olives,olive,olive_trees,orchard,
+Oranges,orange,orange_trees,orchard,
+Almonds- with shell,almond,almond_trees,orchard,
+Apricots,apricot,apricot_trees,orchard,
+Chestnut,chestnut,chestnut_trees,orchard,
+Hazelnuts- with shell,hazelnut,hazel_plants,orchard,
+Pistachios,pistachio,pistachio_trees,orchard,
+Peaches and nectarines,peach,peach_trees,orchard,"Wiki-EN: “Peach and nectarines are the same species, even though they are regarded commercially as different fruits.”"
+Persimmons,persimmon,persimmon_trees,orchard,
+Plums and sloes,plum,plum_trees,orchard,"A hybrid, basically the same, does not matter here anyway"
+Rubber- natural,rubber,rubber_trees,orchard,
+Oil- palm,palm_oil,oil_palms,orchard,
+Oil- palm fruit,palm_oil,oil_palms,orchard,
+Oilseeds,palm_oil,oil_palms,orchard,
+Palm kernels,palm_oil,oil_palms,orchard,usually the “palm cake” is only a side product of palm oil production; tagging them differently or as “palms” (unspecific) could be confusing
+Coconuts,coconut,coconut_palms,orchard,
+Dates,date,date_palms,orchard,
+Blueberries,blueberry,,orchard,
+???????,bayberry,,orchard,
+???????,blackberry,,orchard,
+Raspberries,raspberry,,orchard,
+Strawberries,strawberry,,orchard,
+???????,coca_leaf,coca_plants,orchard,
+Cocoa- beans,cacao,cacao_trees,orchard,"Note different spelling (cf.: trees=cacao_trees); both spellings possible, but tag used here is more common in OSM"
+Pepper (piper spp.),pepper,piper_plants,orchard,
+???????,nopal,nopal_plants,orchard,
+???????,tamarind,tamarind_trees,orchard,
+Pineapples,pineapple,pineapple_plants,orchard,incorrect: trees=ananas_plants
+???????,,shal_trees,orchard,https://en.wikipedia.org/wiki/Shorea_robusta
+???????,blackcurrant,blackcurrant_plants,orchard,
+Tangerines- mandarins- clementines- satsumas,orange,orange_trees,orchard,could also be mapped each one under: trees=mandarin_trees
+???????,lychee,lychee_trees,orchard,
+???????,sand_pear,sand_pear_trees,orchard,
+???????,,eucalyptus_trees,orchard,
+???????,pitaya,pitaya_plants,orchard,
+???????,cherimoya,cherimoya_trees,orchard,
+???????,pomegranate,pomegranate_trees,orchard,
+???????,pecan_nut,pecan_trees,orchard,
+# CURRENTLY MISSING/UNOFFICIAL/PROPOSALS BELOW,,,orchard,
+Chillies and peppers- green,chilli,pepper_plants,orchard,
+Chillies and peppers- dry,chilli,pepper_plants,orchard,
```

Comment:
chilli is possible, at least according to LEO. But with one l seems to be more common. I'll change that.